app/lib/constituency_api.rb

```diff
@@ -1,25 +1,32 @@
 module ConstituencyApi
   ConstituencyApiError = Class.new(RuntimeError)
```

Comment:
why define it this way instead of the more idiomatic:

``` ruby
module ConstituencyApi
  class ConstituencyApiError < RuntimeError; end
end
```

also you could lose the prefix as well:

``` ruby
module ConstituencyApi
  class Error < RuntimeError; end
end
```

then any external rescues can use the shorter `ConstituencyApi::Error`.